File: app/src/main/java/com/example/inscryption/Glavi.java
package com.example.inscryption;

import android.content.Intent;
import android.os.Bundle;
import android.widget.ImageButton;
import androidx.appcompat.app.AppCompatActivity;

import com.example.inscryption.bosses.Angeler;
import com.example.inscryption.bosses.Prospector;
import com.example.inscryption.bosses.Trader;

public class Glavi extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_glavs);

        ImageButton prospectorButton = findViewById(R.id.prospectorButton);
        ImageButton traderButton = findViewById(R.id.traderButton);
        ImageButton anglerButton = findViewById(R.id.anglerButton);

        prospectorButton.setOnClickListener(v -> {
            Intent starIntent = new Intent(this, Prospector.class);
            starIntent.addFlags(Intent.FLAG_ACTIVITY_NO_HISTORY);
            startActivity(starIntent);
        });

        traderButton.setOnClickListener(v -> {
            Intent traderIntent = new Intent(this, Trader.class);
            traderIntent.addFlags(Intent.FLAG_ACTIVITY_NO_HISTORY);
            startActivity(traderIntent);
        });

        anglerButton.setOnClickListener(v -> {
            Intent anglerIntent = new Intent(this, Angeler.class);
            anglerIntent.addFlags(Intent.FLAG_ACTIVITY_NO_HISTORY);
            startActivity(anglerIntent);
        });
    }
}

File: app/src/main/java/com/example/inscryption/kotlnDepreceded/Glavi.kt
package com.example.inscryption.kotlnDepreceded

import android.content.Intent
import android.os.Bundle
import android.widget.ImageButton
import androidx.appcompat.app.AppCompatActivity
import com.example.inscryption.R

class Glavi: AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_glavs)

        val button1: ImageButton = findViewById(R.id.imageButton)
        val button2: ImageButton = findViewById(R.id.imageButton2)
        val button3: ImageButton = findViewById(R.id.imageButton3)


        button1.setOnClickListener {
            val intent1 = Intent(this, Staratel::class.java)
            intent1.addFlags(Intent.FLAG_ACTIVITY_NO_HISTORY)
            startActivity(intent1)
        }

        button2.setOnClickListener {
            val intent2 = Intent(this, Trader::class.java)
            intent2.addFlags(Intent.FLAG_ACTIVITY_NO_HISTORY)
            startActivity(intent2)
        }

        button3.setOnClickListener {
            val intent3 = Intent(this, Angeler::class.java)
            intent3.addFlags(Intent.FLAG_ACTIVITY_NO_HISTORY)
            startActivity(intent3)
        }

    }
}